cli/version_test.go

```diff
@@ -59,11 +59,11 @@ func TestVersionJSON(t *testing.T) {
 	{
 		"release": "",
 		"commit": "",
-		"commitdate": "",
```

Comment:
question: These look very deliberately lower-case, are you really sure you want to change this?

@nasdf @fredcarle

Reply:
When I first implemented the version stuff (JSON and non-JSON) , I don't remember this being a deliberate hard requirement. Is odd to read all lower IMO.

Reply:
I'm ok with the camelCasing.